tools/internal_ci/helper_scripts/prepare_build_macos_interop_rc

```diff
@@ -17,17 +17,6 @@
 # builds. This rc script must be used in the root directory of gRPC
 # and is expected to be used before prepare_build_macos_rc
 
-export CONFIG=opt
-
-# Move gRPC repo to directory that Docker for Mac has drive access to
-mkdir /Users/kbuilder/workspace
```

Comment:
this was causing the "out of disk space" error:   kokoro workers have one small system volume and one large volume mounted to /tmpfs where the build normally happens. Building under /Users/kbuilder we end up having only 2.8G of free space (and the gRPC C++ build is large)